### PR TITLE
Rem 236

### PR DESCRIPTION
**Pull request by Alinadimmokdad**
Sites page

### CODE REVIEW

**Comment by NoraBitarSM** [comment]
functionality tested

File: src/pages/sites/forms/SitesForm.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+      siteGroupId: '',
+      isInactive: false,
+      allowNegativeQty: false
+    },
+    maxAccess,
+    enableReinitialize: true,
+    validateOnChange: true,
+
+    validationSchema: yup.object({
+      name: yup.string().required(),
```
Comment (omarZrayka23) [inline src/pages/sites/forms/SitesForm.js:44]: work?!    yup.string().required(' '),

**Review by omarZrayka23 — CHANGES_REQUESTED** [review]

File: src/pages/sites/forms/SitesForm.js (hunk 1 above)
Comment (Alinadimmokdad) [reply src/pages/sites/forms/SitesForm.js]: done

File: src/components/Shared/GridToolbar.js [hunk 2]
```diff
@@ -20,12 +26,59 @@ const GridToolbar = ({
...
+              assembly: 'ArgusRPT.dll'
+            }))
+            setReportStore(formattedReports)
+            if (formattedReports.length > 0) {
+              setSelectedReport(formattedReports[0])
+            }
+          }
+        })
+        .catch(error => {
+          console.error(error)
```
Comment (omarZrayka23) [inline src/components/Shared/GridToolbar.js:78]: remove console.log

**Review by omarZrayka23 — CHANGES_REQUESTED** [review]

File: src/pages/cities/Forms/CityForm.js [hunk 3]
```diff
@@ -52,6 +52,10 @@ export default function CityForm({ labels, recordId, maxAccess }) {
 
       if (!recordId) {
         toast.success('Record Added Successfully')
+        formik.setValues({
```
[the remark this answers is not in the document]
Comment (omarZrayka23) [reply src/pages/cities/Forms/CityForm.js:55]: ?

**Pull request merged**